app/konnect/runtime-manager/index.md

```diff
@@ -47,6 +45,92 @@ choose to manually configure runtime instances on the following:
 
 See the [runtime instance installation options](/konnect/runtime-manager/runtime-instances/) for more detail.
 
+### Runtime instances dashboard
+
+The following describes the content of the Runtime Instances table found on the landing page of a runtime group.
+You can sort the table by the `Last Ping` column.
+
+SCREENSHOT HERE
+
+Column | Description
+-------|-------------
+Host | The hostname of the instance.
+Type | The runtime instance type. Currently, only {{site.base_gateway}} is supported.
+Last Ping | The last time that this instance received a configuration update from the {{site.konnect_short_name}} control plane to proxy a service.
+Version | The {{site.base_gateway}} version that this instance is running.
+ID | The UUID of the instance.
```

Comment:
I don't know if this really adds anything at all because of how basic the information is.